http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -358,6 +359,243 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
 
     } yield ()
 
+  override def updateAndPollApp(params: UpdateAKSAppParams)(implicit ev: Ask[F, AppContext]): F[Unit] = {
```

Comment:
this method seems like the definition of unscalable... maybe  there's a better encapsulation scheme? I'm imagine something like a mini-interp/object for each specific app with a method to generate the `chartOverrideValues` fr that specific app. That way, instead of a massive case match here, the code would read something like `buildChartOverridesGenerator(app.appType).get()`.

Reply:
Once we are in the audit freezing window, I am totally down to refactor this